docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2910,6 +2910,479 @@ have not been included. These are:
+
+.. todo::
+
+  Add other "keyboard errors", likely ones that swap letters (teh instead of the). This would ideally be in the same typographic function. 
+
+**Fake Names**
+
+For a variety of reasons, some repondents might choose to use a fake name rather 
+than their real name on official forms. To account for this, first select the sample 
+to have noise added. Then for everyone selected, replace their name with a random 
+selection from the `list of fake names here <https://github.com/ihmeuw/vivarium_research_prl/blob/main/noise/2022_04_20c_prl_fake_names.ipynb>`_. 
+Please note that the list is separated into first and last names. 
+
+Limitations: 
+
+- Many of the fake first names include some information about the simulant (daughter, child f, minor) all specify something about the simulant. We will not try to match this information, which might lead to illogical information (an older man being labeled as 'daughter') but will not impact PRL. 
+- Someone who is likely to use a fake name might well do so across multiple observers. This would likely increase PRL challenges but will not be included here. 
+
+**Nicknames**
+
+Many people choose to use nicknames instead of their "real" names. A common example is an 
+Alexander who chooses to go by Alex. These individuals might write their nicknames on forms 
+which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/ihmeuw/vivarium_research_prl/blob/main/noise/nicknames.csv>`_. This was obtained from this `github repo <https://github.com/carltonnorthern/nicknames>`_. 
+
+Only those simulants with names in the csv above are eligible to recieve a nickname. First, 
+determine who is eligible for a nickname. Then select simulants for noise. Lastly, replace their 
+name with any of the nicknames included in the csv. If there are multiple options, 
+select at random. 
+
+Limitations: 
+
+- The list of nicknames above is hand-curated and almost definitely misses some nicknames. It is also likely that it misses nicknames in a bias way based on language of origin, personal history, etc. 
+
+.. note::
+
+  If the process of determining elgibility prior to selecting simulants for noise is challenging, we can work on finding a simpler approach. 
+
+
+**Numeric Miswriting** 
+
+To implement this, select the strings eligible for noise and apply 
+the numeric miswriting noise function to all strings with the user defined 
+character error rate. 
+
+Limitations: 
+
+- This might lead to illogical data, especially for age and dates (e.g., a person who's birthday is 12/87/2000). It is more likely that someone lists an incorrect but still possible birthday/age. However, since the main goal is noise for PRL, we think this is still acceptable. 
+
+.. todo::
+
+  Decide if any numerical strings need to be further refined in approach. Example is zip code where there would be an option to separate the first 3 and last 2 digits. 
```

Comment:
It would probably be good to add links to the [noise package](https://github.com/ihmeuw/vivarium_research_prl/tree/main/src/vivarium_research_prl/noise) and [corruption module](https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/corruption.py) I'm writing for the case studies. For example, I have implemented a [noise function for zipcodes](https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/corruption.py#L172) that take 3 probabilities as input (for the first 2 digits, 3rd digit, last 2 digits) and a [function to swap month and day](https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/corruption.py#L143), as well as functions for incorrect selection and to make some values missing.

Reply:
Thanks!